ert_shared/__init__.py

```diff
@@ -19,3 +23,28 @@ def clear_global_state():
     ERT._implementation = None
     ERT._enkf_facade = None
     ERT = None
+
+class ErtStorage():
+    def __init__(self):
+        self.rdb_url = None
+        self.blob_url = None
+    
+    def initialize(self, rdb_url="sqlite:///entities.db", blob_url="sqlite:///blobs.db"):
+        self.rdb_url = rdb_url
+        self.blob_url = blob_url
+        rdb_engine = create_engine(rdb_url)
+        blob_engine = create_engine(blob_url)    
+        self._upgrade(rdb_engine=rdb_engine, blob_engine=blob_engine)
+            
+        self.RdbSession = sessionmaker(bind=rdb_engine)
+        self.BlobSession = sessionmaker(bind=blob_engine)   
+            
+    def _upgrade(self, rdb_engine, blob_engine, pragma_foreign_keys=True):               
+        if pragma_foreign_keys:
```

Comment:
at this location i will introduce the revision stuff - but currently i just kept the `create_all` as it was .

Reply:
So this PR doesn't have the migration stuff?

Reply:
No - not the migration stuff - i first want to have this one in - and then append the migrations because i've changed spelling and naming of constraints and such in the models along with some cleanup to follow guidelines... i thought it is a natural separation between fixing the session / connection stuff and adding revision system / improving the models.